src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { SortTypes } from '../../types/SortTypes';
+import { Nav } from '../Nav';
+
+interface Props {
+  todos: Todo[];
+  onChangeFilter: (filter: SortTypes) => void;
+  activeFilter: SortTypes;
+}
+
+export const Footer: FC<Props> = ({ todos, onChangeFilter, activeFilter }) => {
+  return (
+    <footer className="todoapp__footer">
```

Comment:
Hide the Footer component if there are no todos.

Reply:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/49568391/f57b8ad6-7e02-4e80-937a-16b535473071)
Like this?